### PR TITLE
DO NOT MERGE

### PR DESCRIPTION
**Pull request by sophistifunk**
NO NOT MERGE

This is me trying to narrow down the cause of a bad CI build

### CODE REVIEW

**Comment by sophistifunk** [comment]
One green...

**Comment by michaelneale** [comment]
SO DON'T MERGE THIS ONE YOU SAY?

**Comment by sophistifunk** [comment]
I gotta spell things out for you freaks

trollface

**Comment by sophistifunk** [comment]
Another green again, giggity

**Comment by cliffmeyers** [comment]
@sophistifunk thanks for looking into this. So this has the same code changes as #1435 but is passing ok?

**Comment by sophistifunk** [comment]
Not all of them yet

**Comment by michaelneale** [comment]
I bet my life it is https://github.com/jenkinsci/blueocean-plugin/pull/1455/commits/8921d56895b40e739a8c30818f44b8a46200f3fe#diff-e92186125148b00ae3b245d3991a7be7R137 that does this. 

the "boolan" -> "boolean" fix. No idea why.. just my gut telling me.

**Comment by sophistifunk** [comment]
Right, testing with ActivityDetailsRow from master, only with the proptypes typo fixed

**Comment by sophistifunk** [comment]
Testing again with just the second call to `buildRunDetailsUrl`

**Comment by sophistifunk** [comment]
*sigh* mystery-meat `Coverage checks have not been met` failure in pipeline-impl, I'm going to start it again and go outside.

File: blueocean-dashboard/src/main/js/components/ActivityDetailsRow.jsx [hunk 1]
```diff
@@ -89,7 +89,7 @@ class ActivityDetailsRow extends Component {
                 <TableCell linkTo={runDetailsUrl}><RunIdCell run={run} /></TableCell>
                 <TableCell linkTo={runDetailsUrl}><CommitId commitId={run.commitId} /></TableCell>
                 { isMultibranch && <TableCell linkTo={runDetailsUrl}>{branchName}</TableCell> }
-                <TableCell><RunMessageCell linkTo={runDetailsUrl} run={run} t={t} /></TableCell>
+                <TableCell><RunMessageCell linkTo={runDetailsUrl} run={run} t={t} changesUrl={changesUrl}/></TableCell>
```
Comment (michaelneale) [inline blueocean-dashboard/src/main/js/components/ActivityDetailsRow.jsx]: @kzantow guesses the lack of space would be a lint fail - which would trip this up lol (NO BUNDLE FOR YOU!)

Comment (kzantow) [reply blueocean-dashboard/src/main/js/components/ActivityDetailsRow.jsx]: yes, confirmed that it's a lint error in the logs and failed `mvntest`

**Comment by michaelneale** [comment]
ok @sophistifunk @kzantow pointed out is a link failure - but that isn't failing the build. So ither it was this change: https://github.com/jenkinsci/blueocean-plugin/commit/403358b4abdee43362a4fedde93a9cb43badf126#diff-58231b16fdee45a03a4ee3cf94a9f2c3 or was a change to parent pom that changed how tests are run and lint failing things... 

this is good stuff

**Pull request closed without merging**